commands/channel_test.go

```diff
@@ -2176,3 +2176,169 @@ func (s *MmctlUnitTestSuite) TestRenameChannelCmd() {
 		s.Require().Equal(printer.GetLines()[0], updatedChannel)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestRemoveChannelCmd() {
```

Comment:
Yes, right, I agree with you and actually it was a bit confusing to me. But I named it like that because on `mattermost-server` repo ([here](https://github.com/mattermost/mattermost-server/blob/master/cmd/mattermost/commands/channel_test.go#L32)) I saw that the test function is called `TestRemoveChannel`... But I'll proceed with the suggestion and change it it that's okay.